src/modules/date/date-string.transformations.ts

```diff
@@ -43,4 +43,9 @@ export class DateStringTransformations {
   getDisplayDateFromDate(date: Date): string {
     return new Intl.DateTimeFormat('en-GB').format(date);
   }
+
+  getDayFromDateOnlyString(dateOnlyString: DateOnlyString): number {
+    const date = new Date(dateOnlyString);
+    return date.getDate();
```

Comment:
I had a look into timezone risks here: `new Date` will interpret `dateOnlyString` as being UTC and then `getDate` getting the day number in the _local_ timezone of that date (which in this case would still be UTC, not the server's local timezone...) - so this doesn't have a timezone issue fortunately!